feat: better brp timout error handling

marnixdessing opened the pull request:
Fixes #583
- Betere foutafhandeling bij timeouts
- Dependency upgrades

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7851140316

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #530 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7858363028

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #531 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7865705660

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #532 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7879785747

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #533 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7894556260

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #534 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: upgrade to new apigatewayv2 packages

marnixdessing committed:
feat: use new apiclient package

WietekeNijmegen reviewed (dismissed):
Moest even goed kijken wat de indents nou deden bij de ifs, maar ziet er goed uit naar mijn idee.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7909175194

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #536 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: update dashboard queries to be more accurate

marnixdessing committed:
fix: pin aws4-axios package on 3.3.0

marnixdessing committed:
Revert "chore(deps): upgrade dependencies"

This reverts commit 6644e52caa64e2876bd767559651c8df9ebb97ec.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7924071987

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #537 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7937808593

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #538 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7945238178

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #539 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7952777421

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #540 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7966613409

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #541 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7981941444

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #542 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/7997449870

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #543 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8012811185

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #544 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8026695374

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #545 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8034179686

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #546 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8041827834

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #547 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8057535250

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #548 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8073221410

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #549 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8088970002

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #550 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8104510977

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #551 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8118819826

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #552 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8126409338

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #553 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8134111746

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #554 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8148954172

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #555 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8164906328

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #556 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8180711828

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #557 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8196694228

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #558 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8210613633

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #559 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8218183683

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #560 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8225833749

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #561 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8241560845

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #562 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Add issuer endpoint to cloudfront (#563)

In a later stage we can use this endpoint to
connect to the issuer, instead of directly. This
allows for logging, and sending custom error
responses.

joostvanderborg committed:
feat: Use cloudfront URL for yivi api connection (#565)

Use cloudfront for yivi issuer api, so we can
use the cloudfront error responses.

Required for #564

joostvanderborg committed:
fix: Directly use host, no param

joostvanderborg committed:
feat: Allow all methods to issue server

Add irma-authorization header to allowed headers.

joostvanderborg committed:
fix: Allow irma-authorization header

Also handles host smarter: Allowed with or without
protocol (https://)

joostvanderborg committed:
Send all headers to origin

joostvanderborg committed:
fix: Set frame-ancestors to none in csp-header

Required to conform to Digid requirement U/PW.03.

joostvanderborg committed:
fix: Name collision between API Gateway and cloudfront origin

joostvanderborg committed:
fix: Reset to direct connection

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8257428960

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #566 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Revert cloudfront changes (#568)

No longer necessary, issue solved in dependency.
Fixes #567

joostvanderborg committed:
feat: revert cloudfront changes (#569)


* fix: tests

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8273442803

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #571 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8289357051

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #572 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8303816135

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #573 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8319382359

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #574 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8335493278

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #575 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8351920441

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #576 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8367685789

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #577 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8383496808

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #578 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8397931770

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #579 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8413212225

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #580 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8428819256

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #581 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/8444661660

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #582 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
docs: disclosure flow simple

marnixdessing committed:
docs: removed single sessionid from plaat

marnixdessing committed:
docs: added description

marnixdessing committed:
Merge branch 'production' into acceptance

joostvanderborg commented on .projenrc.js:
Als deze gefixt is unpinnen.

joostvanderborg reviewed (approved):
Cloudfront-code wordt gecheckt door Marnix, overige gaat over betere timeout-errors, staat al een tijd in acceptatie. Melding over unpinnen van versie van `aws4-axios` als de fix daar in zit gemaakt.
